### PR TITLE
lined up everything pretty good

### PR DESCRIPTION
**Pull request by movingelectrons**
whoops forgot to update the screenshot

### CODE REVIEW

**Comment by Varunram** [comment]
could you upload a screenshot here for reference? Thanks!

**Comment by Varunram** [comment]
Also, git throws this error:
```
 * branch            master     -> FETCH_HEAD
fatal: refusing to merge unrelated histories
```
which means you'd have to replay the commits on current master (I can't seem to cherrypick or rebase, so I guess you should pull from master and then apply your changes manually on top of it)

**Comment by Varunram** [comment]
Closed in favour of #12 (rebased branch). Please change the commit author and then PR from your branch and we should be good.

**Comment by movingelectrons** [comment]
updated screenshot.

meaning I need to pull from Musicoins master, and not my branch, right?

**Review by Varunram — CHANGES_REQUESTED** [review]
Cannot merge unrelated histories.

**Comment by Varunram** [comment]
As I mentioned in the other PR, I can't merge u related histories since that would break all future commits. You need to  rebase on top of current master as is so that all the commits have a shared history, which would make `git diff` work without any trouble (GitHub's interface has fe w problems, `git diff` is more reliable). Also, merging unrelated histories will confuse git on what history to build on top of, so this error would continue propagating as we go down further.

TL;DR: build your changes on top of `master` or pick commits from https://github.com/Musicoin/desktop-interface/pull/11/commits/cc6340c60ca3487d23b76ca035931134582243b5 and build on `rebased`.
